### PR TITLE
Add utt2spk_to_spk2utt

### PR DESCRIPTION
**Pull request by chaisz19**

### CODE REVIEW

File: utils/ls_md5.sh [hunk 1]
```diff
@@ -17,9 +20,8 @@ fi
 gigaspeech_dataset_dir=$1
 
 if ! which jq >/dev/null; then
-  >&2 echo "$0: You have to get jq installed in order to use this. See"
-  >&2 echo "$0: utils/install_jq.sh"
-  exit 1
+  echo "$0: You have to get jq installed in order to use this."
+  utils/install_jq.sh || exit 1
```
Comment (chenguoguo) [inline utils/ls_md5.sh:24]: Add one more line?

```
echo "$0: Attempting jq installation ..."
```

File: toolkits/kaldi/utt2spk_to_spk2utt.pl [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env perl
```
Comment (chenguoguo) [inline toolkits/kaldi/utt2spk_to_spk2utt.pl:1]: Shall we move it to utils/? And then update the calling script as well?

**Review by chenguoguo — CHANGES_REQUESTED** [review]
Overlooked this PR...

**Comment by dophist** [comment]
Is this PR alive ?

**Comment by dophist** [comment]
Is this PR alive ?

**Pull request closed without merging**